src/collectors/github_rate_limit.rs

```diff
@@ -54,14 +56,17 @@ impl GitHubRateLimit {
             .map(|v| v.trim().to_string())
             .collect();
 
-        let users = Self::get_users_for_tokens(tokens).await;
-        let descriptions = Vec::new();
-
-        let rv = Self {
-            users,
-            descriptions,
+        let users = match Self::get_users_for_tokens(tokens).await {
+            Ok(v) => v,
+            Err(e) => {
```

Comment:
Hmm, I'd just propagate the error in this case, maybe adding "unable to get usernames for rate limit stats" as context. I don't want this to silently fail.